cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -163,13 +161,112 @@ impl CatalogQoS {
     }
 }
 
+#[derive(Debug, Clone, Default, Eq, PartialEq)]
+/// TODO
```

Comment:
Forgotten TODO?

Reply:
Yeah, I ended up putting the docs on the variants instead.